lib/ansible/modules/windows/win_chocolatey.ps1

```diff
@@ -207,6 +211,16 @@ Function Choco-Upgrade
         $options += "--source", $source
     }
 
+    if ($user)
+    {
+        $options += "--user=`"'$user'`""
```

Comment:
Why are you double quoting the user and password value, surely only 1 set of quotes is required?